src/constants/theme/colors.ts

```diff
@@ -1,29 +1,29 @@
 export const COLORS = {
-  lunesBlack: '#000e38',
-  lunesWhite: '#f9fafb',
-  lunesGreyMedium: '#5b616a',
-  lunesGreyDark: '#3a4049',
+  primary: '#000e38',
   white: '#ffffff',
-  lunesBlackUltralight: '#e0e4ed',
-  lunesFunctionalIncorrectDark: '#ff5252',
-  lunesRedLight: '#f57f7a',
-  lunesGreenLight: '#7ae6d8',
-  lunesGreenMedium: '#20c5b4',
-  lunesBlueLight: '#97aef1',
-  lunesOrangeLight: '#ffbb4a',
+  background: '#f9fafb',
+  textColor: '#3a4049',
+  textSecondaryColor: '#5b616a',
+  placeholder: '#777e91',
+  disabled: '#e0e4ed',
+  secondarySelectedColor: '#f57f7a',
+  exerciseProgressIndicator: '#20c5b4',
+  containedButtonSelected: '#3d4662',
+  overlay: 'rgba(0, 14, 56, 0.8)',
+  functionalCorrect: '#0adb75',
+  functionalAlmostCorrect: '#ffbb4a',
+  functionalIncorrect: '#ff5252',
+  audioIconHighlight: '#F1635F',
+  audioIconSelected: '#ffa3a3',
+  shadow: 'rgba(0, 0, 0, 0.6)',
   lunesArtikelDas: '#72f399',
   lunesArtikelDiePlural: '#eee12d',
   lunesArtikelDie: '#faa7a7',
   lunesArtikelDer: '#8cc8f3',
```

Comment:
should I remove the prefix?

Reply:
yes and also `article` in english, and maybe for the suffix -masculine -feminine...